psi4/src/export_mints.cc

```diff
@@ -237,7 +237,7 @@ std::shared_ptr<Molecule> from_dict(py::dict molrec) {
     std::vector<double> elez = molrec["elez"].cast<std::vector<double>>();
     std::vector<std::string> elem = molrec["elem"].cast<std::vector<std::string>>();
     std::vector<double> mass = molrec["mass"].cast<std::vector<double>>();
-    std::vector<int> real = molrec["real"].cast<std::vector<int>>();
+    std::vector<bool> real = molrec["real"].cast<std::vector<bool>>();
```

Comment:
Just a note for the future here - `std::vector<bool>` is usually implemented by storing each element as a single bit, so pointer arithmetic and thread safety do not work as expected.  I don't see any usage in this file that would be a problem, but some really evil bugs can possibly result from this kind of change (I found out the hard way, hence this note!)

Reply:
That's good to know. The change here is only to get rid of the `np.bool_` as index warning. I don't see the use of the `real` as becoming more extensive, so your safety analysis will probably continue sound. The only alternative I see is adding a python wrapper fn to do the casting before it heads c-side.